en/docs/setup/migrating-data-publishers.md

```diff
@@ -43,7 +43,7 @@ approach in versions prior to WSO2 IS 5.8.0 and the new design approach.
 This section guides you through migrating an existing data publisher to
 an event handler.
 
-### Migrating data publisher to event handler
+### **Migrating data publisher to event handler**
```

Comment:
Use heading 2 (##) without boldification.

Reply:
There are two reasons not to use H2.
-  The size of the letters is bigger than the others. Therefore it looks a bit abnormal.
-  H2 does not highlight the topic as expected. 

Therefore, using H3 suits for the purpose.

Reply:
Then it breaks the consistency.

Reply:
In most all of the docs H3 is used instead of H2. This needs to be fixed for every doc so that it will be easier for the reader to read.

Fixed: 755e715a1bdd5ac9f01e6a73d74dc83761780cad